packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts

```diff
@@ -1,11 +1,11 @@
 import 'jest-canvas-mock';
+import { getFaceMatchStateInLivenessOval } from '../getFaceMatchState';
```

Comment:
Prefer a corresponding test file for `getFaceMatchStateInLivenessOval` as a separation of concerns